docs/configuring.md

```diff
@@ -0,0 +1,241 @@
+
+- **HEAP64** (minimum 512 M), which should include **HEAPPOOLS64**.
+- **STACK64** (3 MB multiplied by the expected number of concurrent threads)
+- `-Xmx` largest expected VM heap size.
+- The JIT data cache maximum size.
+- The JIT code cache maximum size, if **RMODE64** is supported.
+
+The following guides are available to help you configure Language Environment runtime options and callable services:
+
+- See [z/OS Language Environment Programming Guide](https://www.ibm.com/support/knowledgecenter/SSLTBW_2.4.0/com.ibm.zos.v2r4.ceea200/abstract.htm) for guidance on how to override the default options. Use **RPTOPTS (ON)** to write the options that are in effect to *stderr* on termination.
+- See [z/OS Language Environment Programming Reference](https://www.ibm.com/support/knowledgecenter/SSLTBW_2.4.0/com.ibm.zos.v2r4.ceea300/abstract.htm) for a full list of the available runtime options.
+- See [z/OS Language Environment Debugging Guide](https://www.ibm.com/support/knowledgecenter/SSLTBW_2.4.0/com.ibm.zos.v2r4.ceea100/abstract.htm) for tuning guidance by using **RPTSTG (ON)**.
+
+<i class="fa fa-exclamation-triangle" aria-hidden="true"></i> **Warning:** Changing the runtime options can often degrade performance.
+
+## Configuring large page memory allocation
+
+If your application allocates a large amount of memory and frequently accesses that memory, you might be able to improve performance by enabling large page support on your system.
+
+Large page support is enabled by setting the [`-Xlp:objectheap`](xlpobjectheap.md) and [`-Xlp:codecache`](xlpcodecache.md) options on the command line when you start your application. These options have the following effects:
+
+- The `-Xlp:objectheap` option requests that the Java object heap is allocated by using large pages.
+- The `-Xlp:codecache` option requests that the JIT code cache is allocated by using large pages.
+
+You must also enable large pages on your local system. This process differs according to the operating system.
+
+### AIX systems
+
+AIX supports large page sizes of 64 KB and 16 MB, and a huge page size of 16 GB depending on the underlying system P hardware. To determine which page sizes are supported on a particular system, run `pagesize -a`.
+
+To use large pages to back an application's data and heap segments, specify the **LDR_CNTRL** environment variable. You can set different page sizes for different purposes. The following variables can be used:
+
+- **TEXTPSIZE**: Page size to use for text
+- **STACKPSIZE**: Page size to use for stacks
+- **DATAPSIZE**: Page size to use for native data or HEAP64
+
+The following example sets 4 KB for text and 64 KB for stack, native data, and heap areas:
+
+```
+LDR_CNTRL=TEXTPSIZE=4K@STACKPSIZE=64K@DATAPSIZE=64K
+```
+
+The OpenJ9 VM supports large page usage only to back shared memory segments. Large pages are allocated by using `shmget()` with the **SHM_LGPG** and **SHM_PIN** flags.
```

Comment:
Not sure if we need to talk about these implementation details here. 
Plus, I didn't see SHM_LGPG but SHM_LGPAGE.

Reply:
Removed the sentence. We used to talk about this in the J9 docs, but I did wonder why.